content/chat/setup.textile

```diff
@@ -111,13 +150,48 @@ const App = () => {
 };
 ```
 
+```[swift]
+let realtimeOptions = ARTClientOptions()
+realtimeOptions.key = "{{API_KEY}}"
+realtimeOptions.clientId = "<clientId>"
+let realtime = ARTRealtime(options: realtimeOptions)
+let chatClient = DefaultChatClient(realtime: realtime, clientOptions: nil)
+```
+
+```[kotlin]
+import com.ably.chat.ChatClient
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.types.ClientOptions
+
+val realtimeClient = AblyRealtime(
+    ClientOptions().apply {
+        key = "{{API_KEY}}"
+        clientId = "<clientId>"
+    },
+)
+
+val chatClient = ChatClient(realtimeClient)
+```
+
 A "@ClientOptions@":/api/realtime-sdk#client-options object may be passed to the Pub/Sub SDK instance to further customize the connection, however at a minimum you must set an API key and provide a @clientId@ to ensure that the client is "identified":/auth/identified-clients.
```

Comment:
It will be great, if we provide more clarification related to `clientId` usage as per internal discussion -> https://ably-real-time.slack.com/archives/CURL4U2FP/p1733668250689229

Reply:
it's out of the scope of this PR